libs/deps/mas/merch-card.js

```diff
@@ -1735,6 +1736,7 @@ merch-card [slot='heading-s'] {
     line-height: var(--consonant-merch-card-heading-s-line-height);
     margin: 0;
     color: var(--spectrum-gray-800, var(--merch-color-grey-80));
+    font-weight: 700;
```

Comment:
the values below seem to repeat a lot, i wonder if its alrady worth it to put it in common place ('heading-' or smthg)
```
margin: 0;
    color: var(--spectrum-gray-800, var(--merch-color-grey-80));
    font-weight: 700;
```